docs/java-application-development/kura-workspace-setup.md

```diff
@@ -0,0 +1,152 @@
+# Kura Workspace Setup
+
+This document describes how to set up the Kura workspace development environment, which consists of the following components:
+
+- JVM (Java JDK SE 8)
+- Eclipse IDE
+- Kura Workspace
+
+**This setup will allow you to develop applications or bundles running on Kura. It will install only the APIs and the examples.** If you want to contribute to the Kura project follow [this guide](./development-environment-setup.md) instead.
+
+The Kura development environment may be installed on Windows, Linux, or Mac OS. The setup instructions will be the same across each OS though each system may have unique characteristics.
+!!! info
+    The local emulation of Kura code is only supported in Linux and Mac, not in Windows.
+
+## JVM Installation
+
+Download and install JDK SE 8 from the following links as appropriate for your OS.
+
+For Windows and Linux users, the JDK can be downloaded from the following link: [Java SE 8 Downloads](https://adoptium.net/temurin/releases/?version=8).  Use the latest version of Java SE Development Kit and download the version appropriate for your system.
```

Comment:
> version of Java SE Development Kit

version of the Java SE Development Kit

Reply:
Addressed in 5d9d6a54ce8512d1ca35d2d459bcc47798e77cc9